src/main/java/ch/njol/skript/expressions/ExprEntities.java

```diff
@@ -56,19 +57,20 @@
  * @author Peter Güttinger
```

Comment:
You can remove the author tag while you're at it. We don't use those anymore

Reply:
Will do 👍